src/software/thunderscope/robot_diagnostics/chicker.py

```diff
@@ -197,8 +173,12 @@ def send_command(self, command):
         # sends kick, chip, autokick, or autchip primitive
         if command == ChickerCommandMode.KICK:
             power_control.chicker.kick_speed_m_per_s = power_value
+            self.proto_unix_io.send_proto(PowerControl, power_control)
+            power_control.chicker.kick_speed_m_per_s = 0
```

Comment:
The same bug exists for AUTOKICK and AUTOCHIP as well (going from AUTO to NOAUTO).
I would recommend defining a 5th enum type (e.g. `ChickerCommandMode.None`) which just sends an empty proto, not filling in any fields. Or another approach would be having a second `send_proto` call after the one at the end of this function which just sends an empty `PowerControl()` proto.

Could you also add a comment on why we need to reset the proto we send? (something a long the lines of: because cached messages are resent to the robot if we don't send a new PowerControl message, thus we need to override the cached message to avoid spamming kick/chip calls)

Reply:
done